index.html

```diff
@@ -18,8 +18,16 @@
     </nav>
```

Comment:
- [ ] Hi @malembi :wave: , could it be that you are having difficulty removing this `nav` tag around your `<a href="#" class="logo"><p>MALEMBI</p></a>` as suggested by previous code reviewers, or are simply making the change in another branch because I can still see it wrapped around your logo? It is imperative to aim at using semantic tags because they impress recruiters and also increase your site's accessibility. Kindly remove, my good friend :smile:

Reply:
yes i was doing it on the other branch, thanks for that.